types/three/three-core.d.ts

```diff
@@ -6179,14 +6186,17 @@ export class WebGLState {
     };
 
     init(): void;
+    texImage2D(target: number, level: number, internalformat: number,
```

Comment:
Seems this is an indirection of this method https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/texImage2D
Which has a bunch of other overloads, in lieu of adding all the overloads maybe leave a comment linking the API and note the unimplemented overloads?

It's also undocumented on the threejs API, are you quite sure that three intends this to be used from outside? There are other ways to get the gl context and use it directly even if three didn't provide this method.

Reply:
You are right. I needed to indirect through WebGLState in several cases to ensure consistency between three.js and another GL library, but in this case no state information is cached, so I can use WebGL directly. This interface may not be public, but it is already given later in this class declaration with an empty signature. I will add a comment there like you said.

Reply:
That's what I was thinking. It may also be ok to exclude the member entirely from the declaration if it's not meant to be visible to the outside. Either way works for me.